it/uk/gov/hmrc/mobileusercontact/test/IntegrationTestJson.scala

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmrc.mobileusercontact.test
+
+object IntegrationTestJson {
```

Comment:
I think it would be nice to have separate objects for Feedback JSON and Support JSON, rather than one object for all integration test JSON (that runs the risk of getting really big)